db/migrate/20180626235055_create_images.rb

```diff
@@ -0,0 +1,9 @@
+class CreateImages < ActiveRecord::Migration[5.1]
+  def change
+    create_table :images do |t|
+      t.string :link
```

Comment:
By default, columns allow `null` values. Since we don't have a use case where we want an image to be saved without a link, let's add `null: false` as an argument to this line here